test/unittests/commands/ingest_xmlupload/test_apply_ingest_id.py

```diff
@@ -0,0 +1,61 @@
+from lxml import etree
+
+from dsp_tools.commands.ingest_xmlupload.apply_ingest_id import replace_filepath_with_sipi_id
+
+# pylint: disable=missing-class-docstring,missing-function-docstring
+
+
+class TestReplaceBitstreamPaths:
+    def test_all_good(self) -> None:
+        xml = (
+            "<knora>"
+            '<resource label="Fluffy Image" restype=":Image2D" id="Fluffy1" permissions="res-default">'
+            '<bitstream permissions="prop-default">images/Fluffy.jpg</bitstream>'
+            "</resource>"
+            "</knora>"
+        )
```

Comment:
Personally, I prefer this:

```suggestion
        xml = """
        <knora>
            <resource label="Fluffy Image" restype=":Image2D" id="Fluffy1" permissions="res-default">
                <bitstream permissions="prop-default">images/Fluffy.jpg</bitstream>
            </resource>
        </knora>
        """
```